### DISCUSSION TITLE
PyCon Zimbabwe Website

### DISCUSSION THREAD

**Discussion by hbutau** [PyZim/PyZim.github.io#38] [General]
b

**Comment by hbutau** [comment]
Heya @ronnzw @bornie21 @Kudzmat @JustNunuz @cliff688 @tzinadah . I started this discussion so we can start kickstart replacing the Pelican-based website we have been using in the past. Maybe we can switch to another stack? I have also initiated a Kanban board with potential tasks [here](https://github.com/orgs/PyZim/projects/4)

**Reply by cliffordgama** [reply]
(The link with potential tasks is broken.)

**Reply by hbutau** [reply]
> (The link with potential tasks is broken.)

@cliff688 I have fixed it. I had set It to be private.
